README.adoc

```diff
@@ -51,23 +51,27 @@ The plugin provides bootstrap script that creates sample project with:
 * proper source file layout
 * sample https://gatling.io/docs/current/general/simulation_structure/[Simulation class], demonstrating sufficient `Gatling` functionality
 
-For this quickstart - `git` and `curl` must be instlled and available in `$PATH`.
+For this quickstart - `git` and `curl` must be installed and available in `$PATH`.
 
-.Generate sample Gradle project and run Gatling simulation
-[source, bash]
-----
-curl -sL https://git.io/Jf2Uk | bash -s ~/sample-gradle-gatling
-----
 
-.Navigate to new project folder and run all simulations
+[TIP]
+====
+Sample project skeleton should be adjusted after a new plugin will be released.
+====
+
+.Generate sample Gradle project and run Gatling simulation, follow on-screen instructions.
 [source, bash]
 ----
-cd ~/sample-gradle-gatling
-./gradlew gatlingRun
+curl -sL https://git.io/JJj3A | bash -s ~/gradle-gatling-sample
 ----
 
 == Installation
 
+[TIP]
+====
+This should be changed after a new plugin will be released.
+====
```

Comment:
Not in git history.

Reply:
Again, didn't understand the meaning of this comment.

Reply:
removed